openssf-compiler-options.yaml

```diff
@@ -17,7 +17,7 @@ pipeline:
     runs: |
       mkdir -p "${{targets.destdir}}"
       cp -r etc usr "${{targets.destdir}}"
```

Comment:
Needs epoch bump.

Reply:
😡 every time I'm so bad at remembering that.